docs/usage/control_plane_migration.md

```diff
@@ -22,4 +22,33 @@ If the `Shoot` has different `.spec.seedName` and `.status.seedName` a process i
 
 If the process is successful, we update the status of the `Shoot` by setting the `.status.seedName` to the null value. That way, a restoration is triggered in the `Destination Seed` and `.status.lastOperation` is changed to `Restore`. The control plane migration is completed when the `Restore` operation has completed successfully.
 
-By default the shoot's etcd backups will continue to be uploaded to the `BackupBucket` of the `Source Seed` after control plane migration has finished. If you want the etcd backups to instead be uploaded to the `BackupBucket` of the `Destination Seed`, you have to enable the `CopyEtcdBackupsDuringControlPlaneMigration` feature gate on the `gardenlet` by adding the following command flag: `--feature-gates=CopyEtcdBackupsDuringControlPlaneMigration=true`. Note that this will also move the existing etcd backups to the `BackupBucket` of the `Destination Seed` during the control plane migration process.
\ No newline at end of file
+By default the shoot's etcd backups will continue to be uploaded to the `BackupBucket` of the `Source Seed` after control plane migration has finished. If you want the etcd backups to instead be uploaded to the `BackupBucket` of the `Destination Seed`, you have to enable the `CopyEtcdBackupsDuringControlPlaneMigration` feature gate on the `gardenlet` by adding the following command flag: `--feature-gates=CopyEtcdBackupsDuringControlPlaneMigration=true`. Note that this will also move the existing etcd backups to the `BackupBucket` of the `Destination Seed` during the control plane migration process.
+
+## Triggering the migration
+
+For controlplane migration, operators with necessary RBAC can request a [`shoot/binding`](../concepts/scheduler.md#shootsbinding-subresource), with the following commands:
+
+```bash
+export NAMESPACE=my-namespace
+export SHOOT_NAME=my-shoot
+kubectl create \
+    -f <path>/<to>/binding-request.json \
+    --raw /apis/core.gardener.cloud/v1beta1/namespaces/${NAMESPACE}/shoots/${SHOOT_NAME}/binding | jq -r ".status"
+```
+
+Here, the `binding-request.json` has the following content:
+
+```json
+{
+    "apiVersion": "core.gardener.cloud/v1beta1",
+    "kind": "Binding",
+    "metadata": {
+      "name": "<shoot-name>"
```

Comment:
What about the namespace? Don't I need to specify the namespace as well?

Reply:
It will use the $NAMESPACE on line 36.